src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/ScaffoldingServer.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading;
+using Microsoft.VisualStudio.Web.CodeGeneration.Utils.Messaging;
+using Newtonsoft.Json;
+
+namespace Microsoft.VisualStudio.Web.CodeGeneration.Tools
+{
+    internal class ScaffoldingServer : IDisposable, IMessageSender
```

Comment:
I wonder if this could be generalized... (see https://github.com/aspnet/EntityFramework/issues/6128)